bigbluebutton-html5/imports/ui/components/user-list/user-list-graphql/user-participants-title/component.tsx

```
@@ -57,14 +57,18 @@ const UserTitle: React.FC<UserTitleProps> = ({
 
 const UserTitleContainer: React.FC = () => {
   const getCountData = () => {
-    const { data: countData } = useDeduplicatedSubscription(USER_AGGREGATE_COUNT_SUBSCRIPTION);
+    const { data: countData } = useDeduplicatedSubscription<UsersCountSubscriptionResponse>(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)